### ISSUE TITLE
Picklist update failure not caught during validation

### ISSUE THREAD

**Issue by gvpalsson** [flxbl-io/sfp#32]
**Describe the bug**
During the run of the validate command the error "Unable to process Picklist update due to URI malformed" is reported when processing the picklist update for one of our packages. The picklist processing terminates so all picklist fields left to be updated are not processed causing inconsistency in our orgs.

**To Reproduce**
1. Include a picklist with the % character in its API name in a standard unlocked package.
2. Run the sfp validate command to validate the package.
Result: "Unable to process Picklist update due to URI malformed" error is reported in the validation output when that field is being processed by the picklist updater. A successful validation means this bad value will fail the picklist updater to update all picklists when the package is being deployed to orgs.

**Expected behavior**
The sfp validate command should report a failed validation so the picklist updater will not break for package deployments.

**Screenshots**
![image](https://github.com/flxbl-io/sfp/assets/158459078/b40ca438-3fd4-4c23-86c4-53197971016e)

**Platform Details (please complete the following information):**
 - OS: 
 - Version: sfp  -- The DX@Scale CLI -Version:25.5.3 -Release:December 23
 - Salesforce CLI(sfdx cli) Version:  The version used in the DX@Scale image.
 - CI Platform:  Bitbucket cloud

**Additional context**

**Comment by Rocko1204** [comment]
Hi @gvpalsson I have resolved this issue with fix #8 So its included in the release feb 24. Regards Ronny

**Comment by michallachowski** [comment]
@Rocko1204 After an update to v36.0.10 the issue still appears:

![image](https://github.com/flxbl-io/sfp/assets/3834441/d9944fef-9dcf-483f-aba8-366c6c6ec1be)

I did some research around it, and it looks like it happens because of `%` character in picklist API Name and / or label, which is not handled properly in [PicklistEnabler.ts](https://github.com/flxbl-io/sfp/blob/main/packages/sfp-cli/src/core/package/deploymentCustomizers/PicklistEnabler.ts).

Here're changes that I tested already, and it seem to be handling `%` properly without using `decodeURI()`: https://github.com/flxbl-io/sfp/compare/main...michallachowski:sfp:picklist-enabler-decode-uri:
- output of `decodeURIComponent()` is not being used. Do we need to call these methods here?
- do we need to use `decodeURI()` on picklist api name and label? What was the purpose of implementing it in this way?
